brunton_lab_to_nwb/brunton_lab_to_nwb.py

```diff
@@ -203,10 +220,11 @@ def convert_dir(in_dir, n_jobs=1, overwrite: bool = False):
     nwb_files = [x.stem for x in all_files if ".nwb" in x.suffix]
 
     if overwrite:
-        in_files = [str(in_dir / f"{x}.h5") for x in all_data_files]
+        in_files = [str(in_dir + '\\' + f"{x}.h5") for x in all_data_files]
     else:
-        in_files = [str(in_dir / f"{x}.h5") for x in all_data_files if x not in nwb_files]
-    out_files = [str(in_dir / f"{Path(x).stem}.nwb") for x in in_files]
+        print(all_data_files, in_dir)
+        in_files = [str(in_dir + '\\' + f"{x}.h5") for x in all_data_files if x not in nwb_files]
+    out_files = [str(in_dir + '\\' + f"{Path(x).stem}.nwb") for x in in_files]
 
```

Comment:
This looks like a windows vs. linux issue. Use `os.path.join` or `pathlib.Path` objects so this works in any OS

Reply:
ah, sorry about that--I saw the elec_loc_labels.csv in the repo and see it in being loaded directly from the top level in the package so I thought that I should follow suit and include this for the conversion of the entire dataset.  Whoever converts the rest of the files and inserts the event info will need this file and edit the path on line 198 to point to it.

Reply:
you should edit the call signature of `run_conversion` to include the events file